api/openapi-spec/swagger.json

```diff
@@ -15626,21 +15629,24 @@
       "format": "int64"
      },
      "minimumReservePVCBytes": {
+      "description": "MinimumReservePVCBytes is the amount of space, in bytes, to leave unused on disks. Defaults to 131072 (128KiB)",
       "type": "integer",
       "format": "int64"
      },
      "nodeSelectors": {
+      "description": "NodeSelectors allows restricting VMI creation to nodes that match a set of labels. Defaults to none",
```

Comment:
nit: Inconsistent sentence endings. some descriptions end with periods and some don't. I'll mention only this line for brevity but there are others.

Reply:
The idea was to end regular sentences with periods, but not default values (to avoid confusion, like thinking that a default of `1.` means the value is float or something).
However, as you pointed out, even that was inconsistent! I had a quick look and addressed the ones I found. PTAL!